tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -423,13 +597,23 @@ describe('metrics reducers', () => {
         },
       };
 
+      const cardMetadata = {
+        plugin: PluginType.SCALARS,
+        tag: 'tagA',
+        runId: null,
+      };
+      const cardId = getCardId(cardMetadata);
+
       const action = actions.metricsTagMetadataLoaded({tagMetadata});
       const nextState = reducers(beforeState, action);
 
-      expect(nextState.cardMetadataMap['<cardId>']).toEqual(
-        createScalarCardMetadata()
+      const expectedCardMetadataMap: CardMetadataMap = {};
+      expectedCardMetadataMap[cardId] = cardMetadata;
+
+      expect(nextState.cardMetadataMap['<cardId>']).not.toBe(origCardMetadata);
```

Comment:
undefined or null does not work out because it checks type. To be concrete the meesage is
```
 Argument of type 'undefined' is not assignable to parameter of type 'Expected<CardMetadata>'.
```

I replace it with `hasOwnProperty`
```
expect(nextState.cardMetadataMap.hasOwnProperty('<cardId>')).toBe(false);
```